lang/nl.yml

```
@@ -21,9 +21,19 @@ nl:
       one: 'Een thema'
       other: "{count} thema's"
     SINGULARNAME: 'Thema'
+  ElementArticles:
+    Optional: 'Optioneel'
+    ShowHighlightedArticlesOnly: 'Weergeef alleen uitgelichte artikelen'
+    ShowPinnedArticlesAtTop: 'Weergeef vastgeprikte artikelen bovenaan'
+    MaxAmount: 'Maximaal aantal weer te geven artikelen'
+    ShowMoreArticlesButton: "Weergeef 'meer artikelen' knop"
+    ShowMoreArticlesButtonText: "'Meer artikelen' knop tekst"
+    Results: "Resultaten"
```

"resultaten" can be inside single quotes
```suggestion
    Results: 'Resultaten'
```